specification/compute/resource-manager/Microsoft.Compute/stable/2020-09-30/disk.json

```diff
@@ -1647,6 +1650,18 @@
         "tier": {
           "type": "string",
           "description": "Performance tier of the disk (e.g, P4, S10) as described here: https://azure.microsoft.com/en-us/pricing/details/managed-disks/. Does not apply to Ultra disks."
+        },
+        "securityType": {
+          "type": "string",
+          "description": "Specifies the SecurityType of the VM. It is set as SecureBoot for TrustedVMs.",
```

Comment:
This is a property of a disk. The comment should talk about what it means for the disk, not the VM (there may not be any VM if the disk is unattached).

Reply:
SecurityType is used only in context of VM. Can we use description similar to hyperVGeneration?

Reply:
So what is the proposed description? It should say something like "OS disk that can used to create a VM with the specified Security Type"

Reply:
Proposed description is - "Specifies the SecurityType of the VM. Applicable for OS disks only."
I've pushed another iteration to modify this.